linux/ibus-keyman/src/KeymanSystemServiceClient.cpp

```diff
@@ -1,5 +1,10 @@
+#include "config.h"
 #include <cassert>
+#if DBUS_IMPLEMENTATION == SYSTEMD
 #include <systemd/sd-bus.h>
+#else
```

Comment:
Do you want to check `#elif DBUS_IMPLEMENTATION == BASU`?

For all the other .cpp and .h files too...